ts/screens/authentication/IdpSelectionScreen.tsx

```diff
@@ -151,8 +137,4 @@ const IdpSelectionScreen: React.SFC<Props> = props => {
   );
 };
 
-const mapStateToProps = (state: GlobalState) => ({
-  isSessionExpired: isSessionExpiredSelector(state)
-});
-
-export default connect(mapStateToProps)(IdpSelectionScreen);
+export default connect()(IdpSelectionScreen);
```

Comment:
since we are not using anymore redux stuffs we can remove **_connect_**

Reply:
It seems the `connect()` is required if we use the `ReduxProps.dispatch` method, otherwhise we get an error when it is called.
To improve code readability, we could always declare the functions dispatching actions into the `mapDispatchToProps` item.